.gitignore

```diff
@@ -271,4 +271,10 @@ tags
 [._]*.un~
 
 # End of https://www.gitignore.io/api/vim,linux,macos,python,pycharm
-submit_test
\ No newline at end of file
+submit_test
+
+# Custom.
+Pipfile.lock
+requirements.txt_dev
+requirements.txt_stable
```

Comment:
I don't think we need those (see other comment).

Reply:
I'd prefer to remove those "extra" requirements from the .gitignore file.

Reply:
It is ok. However, I would still prefer to keep the lock file.

Reply:
Yes, that's fine.